app/(main)/song/[id]/artist-link.tsx

```
@@ -0,0 +1,24 @@
+ * @returns Promise<JSX.Element> - A promise that resolves to a JSX element
+ * 
+ * @example
+ * const artistLink = await ArtistLink({artistID: "123"});
+ */
+
+export default async function ArtistLink({artistID}: {artistID: string}) {
+    const artist = await fetchArtistById(artistID);
+    if(artist){
+      return (
+        <Link href={`/artist/${artist.id}`}>
+          <a className="text-blue-500 hover:text-blue-700">{artist.name}</a>
```

Biome (1.9.4)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)